src/modules/telemtry/entities/telemetry.ts

```
@@ -0,0 +1,40 @@
+    Entity,
+    PrimaryGeneratedColumn,
+    Column,
+    BaseEntity,
+  } from 'typeorm';
+  
+  @Entity('telemetry')
+  export class Telemetry extends BaseEntity {
+    @PrimaryGeneratedColumn()
+    id: number;
+  
+    @Column({ type: 'bigint', nullable: false })
```

ESLint

[error] Delete `··`

(prettier/prettier)